ott/core/sinkhorn_lr.py

```diff
@@ -30,12 +31,24 @@ class LRSinkhornState(NamedTuple):
   q: Optional[jnp.ndarray] = None
   r: Optional[jnp.ndarray] = None
   g: Optional[jnp.ndarray] = None
+  q_prev: Optional[jnp.ndarray] = None
```

Comment:
Feel like the state is too large, would suggest the following:
- remove `{q,r,g}_prev`, have only `compute_criterion` in `LRSinkhornState` and refactor it as:
  ```python
  def compute_criterion(self, previous_state: "LRSinkhornState") -> float:
    err_1 = ...
    ...
    return err_1 + err_2 + err_3
  ```
- make `criterion: Optional[float] = None` -> `criterions: Optional[jnp.ndarray]` and store the value at each iteration - might be useful to plot the evolution in the future